.babelrc

```diff
@@ -1,11 +1,22 @@
 {
   "presets": [
-    ["env", {
-      "targets": {
-        "browsers": ["> 1%", "last 2 versions", "not ie <= 8"]
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": [
+            "> 1%",
+            "last 2 versions",
+            "not ie <= 8"
```

Comment:
We should change this to not worry about IE 9 and 10.

Reply:
We should, just not here.

Reply:
https://trello.com/c/RGm3Ssw0/282-update-browser-support-in-babel